Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -766,6 +775,10 @@ static void logProcessPoolState(const WebProcessPool& pool)
         "com.apple.system.DirectoryService.InvalidateCache.user"_s,
 #else
         // EMBEDDED_FORWARDED_NOTIFICATIONS
+        "com.apple.ManagedConfiguration.profileListChanged"_s,
+        "com.apple.managedconfiguration.passcodechanged"_s,
+        "com.apple.managedconfiguration.restrictionchanged"_s,
+        "com.apple.managedconfiguration.settingschanged"_s,
```

Comment:
My understanding is these should be synced with EMBEDDED_FORWARDED_NOTIFICATIONS in process-entitlements.sh too, but this patch does not add them there, so it seems wrong.

Also, can we keep this patch to only adding new notifications? It's a bit unclear what notifications are new in this patch...

Reply:
Ah, that's a good catch, will fix!

I will also move the notifications to common.sb in a separate patch, in order to make it more clear what the new notifications are.

Thanks for reviewing!